src/js/builtins/Module.ts

```diff
@@ -3,14 +3,16 @@ export function main() {
   return $requireMap.$get(Bun.main);
 }
 
-$visibility = "Private";
+$visibility = "PrivateRecursive";
+$alwaysInline;
```

Comment:
Adds `@alwaysInline` Like JSC does for `Array#filter`:
https://github.com/WebKit/WebKit/commit/6cb56ee3051f43ee8030b9c34b4ea035a2effa6a